app.js

```diff
@@ -2,16 +2,18 @@ const express = require('express');
 const { getAllTopics } = require('./controllers/topics-controller');
 const { availableEndpoints } = require('./controllers/endpoint-controller');
 const { getArticle } = require('./controllers/articles-controller');
+const { getAllComments } = require('./controllers/comments-controller');
 const app = express();
 
+app.use('/api/articles/:article_id/comments', getAllComments);
 app.use('/api/articles/:article_id', getArticle);
 app.use('/api/topics', getAllTopics);
 app.use('/api', availableEndpoints);
 
 
 app.use((err, request, response, next) => {
   
-  if (err.status === 400) {
+  if (err.status === 400 || err.code === '22P02') {
```

Comment:
We should not combine status and codes like this. Keep your custom errors in one function and your PSQL erros in another. Checking the status or the codes together like this could lead to some errors going to the wrong handler